src/components/02-components/data-table/data-table.data.tsx

```diff
@@ -228,3 +228,36 @@ export const staffFilterModel: GridFilterModel = {
   items: [{ field: 'col3', operator: 'contains', value: 'Education' }],
   logicOperator: GridLogicOperator.Or,
 };
+
+export const enrolledStudentsTableRows = [
+  {
+    id: 1,
+    firstName: 'Emily',
+    lastName: 'Parker',
+    lastlogin: '09/12/2023 - 10:10am',
+    experiences: '#2 - Community Roots Garden',
+  },
+  {
+    id: 2,
+    firstName: 'Mathew',
+    lastName: 'Ross',
+    lastlogin: '03/28/2023 - 11:16am',
+    experiences: '#1 - Community Roots Garden',
+  },
+];
+
+export const enrolledStudentsTableColumns = [
+  { field: 'firstName', headerName: 'Name', width: 200 },
+  { field: 'lastName', headerName: 'Last Name', width: 200 },
+  { field: 'lastlogin', headerName: 'Last login', width: 200 },
+  {
+    field: 'url',
+    headerName: 'Experiences',
+    width: 250,
+    type: 'actions',
+    disableExport: true,
+    renderCell: (params: GridRenderCellParams) => (
+      <Link href={'/course/experience/' + params.row.id}>{params.row.experiences}</Link>
```

Comment:
We might need to discuss this, it seems like this link should be `/course/{course-id}/experience/{experience-id}` so we are probably missing the course id.

The alternative is we have a url like `/experiences/{experience-id}` but currently this url path seems like it needs adjustment.

Reply:
Link changes to `/course/{course-id}/experience/{experience-id}`